src/components/Footer.svelte

```diff
@@ -0,0 +1,29 @@
+<footer>
+    <p>© {new Date().getFullYear()} kavin.rocks</p>
```

Comment:
Why 😂😂😂

Reply:
Because why not? :)
No worries, we can replace it with `GNU/Linux` to be transparent about what kavin.rocks is hosted on of course ^)